src/vm/gccover.cpp

```diff
@@ -1133,7 +1133,11 @@ bool IsGcCoverageInterrupt(LPVOID ip)
     }
 
     // Now it's safe to dereference the IP to check the instruction
+#ifdef _TARGET_ARM_    
+    UINT16 instructionCode = *reinterpret_cast<UINT16 *>(ip);
```

Comment:
I understand that the cast is needed. My comment was whether this  also needs to clear the thumb bit - e.g. using `PCODEToPINSTR`. After taking closer look, there are multiple places in the Unix-specific code in this file that are missing handling of thumb bit - so cleaning it up can be done in separate commit later.